torch/fx/subgraph_rewriter.py

```diff
@@ -217,8 +221,16 @@ def _replace_pattern(
 
     # Get the graphs for `gm`, `pattern`, `replacement`
     original_graph: Graph = gm.graph
-    pattern_graph: Graph = symbolic_trace(pattern).graph
-    replacement_graph: Graph = symbolic_trace(replacement).graph
+
+    if isinstance(pattern, GraphModule):
+        pattern_graph = pattern.graph
+    else:
+        pattern_graph = symbolic_trace(pattern).graph
```

Comment:
if we do symbolic_trace then the pattern can be a `nn.Module` I think?

Reply:
lol... nn.Module is also a Callable, coz it implemented __call__ 
So we are good

Reply:
`GraphModule` is a subclass of `nn.Module` though

Reply:
sorry I think I misread the PR, sounds good